Sources/SwiftSyntax/gyb_generated/syntax_nodes/SyntaxNodes.swift

```diff
@@ -58,6 +58,52 @@ extension MissingSyntax: CustomReflectable {
 /// a CodeBlock.
 /// 
 public struct CodeBlockItemSyntax: SyntaxProtocol, SyntaxHashable {
+  public enum Item: SyntaxProtocol {
+    case `decl`(DeclSyntax)
+    case `stmt`(StmtSyntax)
+    case `expr`(ExprSyntax)
+    case `tokenList`(TokenListSyntax)
+    case `nonEmptyTokenList`(NonEmptyTokenListSyntax)
+    public var _syntaxNode: Syntax {
+      switch self {
+      case .decl(let node): return node._syntaxNode
+      case .stmt(let node): return node._syntaxNode
+      case .expr(let node): return node._syntaxNode
+      case .tokenList(let node): return node._syntaxNode
+      case .nonEmptyTokenList(let node): return node._syntaxNode
+      }
+    }
+    init(_ data: SyntaxData) { self.init(Syntax(data))! }
+    public init(decl node: DeclSyntax) { self = .decl(node) }
```

Comment:
I think it would also be useful to have
```swift
public init<T: DeclSyntaxProtocol>(decl node: T) 
```

so you can initialize `.decl` with any decl node without needing to upcast it to `DeclSyntax` first.